index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>PORTFOLIO</title>
+		<link rel="stylesheet" type="text/css" href="normalize.css">
+		<link rel="stylesheet" type="text/css" href="portfolio.css">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+	</head>
+	<body>
+		<div class="sticky">
+			<nav class="navholder">
+				<div class="contact navbutton"><a class="contacticon" href="#contactme">CONTACT ME</a></div>
+				<div class="portfolio navbutton"><a class="portfolioicon" href="#portfolio">PORTFOLIO</a></div>
+				<div class="about navbutton"><a class="abouticon"  href="#aboutme">ABOUT ME</a></div>
+				<div class="home navbutton"><a class="homeicon" href="#home">HOME</a></div>
+			</nav>
+		</div>
+		<div class="introbox">
+			<div>
+				<h1 id="home">Hello There</h1>
+				<h3>Welcome to my world!</h3>
+				<p>This is my world created with html and css.
+				</p>
+			</div>
+		</div>
+		<h2 id="aboutme">ABOUT ME</h2>
+		<div class="aboutmecontainer">
+			<div class="imgcontainer">
+				<img class ="abmimg " src="Assets/panda.jpg" alt="about me img">
```

Comment:
img is not a word. alt text needs to be English so screen readers can read it out